lms/djangoapps/grades/tasks.py

```diff
@@ -40,17 +42,33 @@
     ValidationError,
     DatabaseNotReadyError,
 )
+POLICY_CHANGE_GRADES_EXPIRY = 60 * 60 * 24
 RECALCULATE_GRADE_DELAY = 2  # in seconds, to prevent excessive _has_db_updated failures. See TNL-6424.
 
 
+def locked_task(expiry_seconds):
```

Comment:
Also, we should be able to write a simple unit test for this.  It doesn't have to call `compute_all_grades_for_course`, (although it might my nice to have such a test).  We can talk this out in person, if you'd like.